UefiTestingPkg/AuditTests/PagingAudit/Windows/PagingReportGenerator.py

```diff
@@ -47,45 +46,46 @@ def Parse(self):
         Page4kbFileList =  glob.glob(os.path.join(self.DatFolderPath, "*4K*.dat"))
         MatFileList =  glob.glob(os.path.join(self.DatFolderPath, "*MAT*.dat"))
         GuardPageFileList =  glob.glob(os.path.join(self.DatFolderPath, "*GuardPage*.dat"))
+        PlatformInfoFileList = glob.glob(os.path.join(self.DatFolderPath, "*PlatformInfo*.dat"))
 
         logging.debug("Found %d Info Files" % len(InfoFileList))
         logging.debug("Found %d 1gb Page Files" % len(Page1gbFileList))
         logging.debug("Found %d 2mb Page Files" % len(Page2mbFileList))
         logging.debug("Found %d 4kb Page Files" % len(Page4kbFileList))
         logging.debug("Found %d MAT Files" % len(MatFileList))
         logging.debug("Found %d GuardPage Files" % len(GuardPageFileList))
-
-        SetArchitecture(self.Architecture)
+        logging.debug("Found %d Platform Info Files" % len(PlatformInfoFileList))
+
+        for info in PlatformInfoFileList:
+            ParsePlatforminfofile(info)
+
+        if Globals.ParsingArchitecture == "":
+            raise Exception("Architecture not Identified in PlatformInfo.dat")
+        elif Globals.ParsingArchitecture == "AARCH64" and Globals.ExecutionLevel == "":
+            logging.error("Architecture is AARCH64 but Execution Level wasn't specified. Assuming EL1")
+            Globals.ExecutionLevel = "EL1"

```

Comment:
Why is EL1 the EL assumed? SBSA spec would have this at EL2, seems to be a reasonable default?

Reply:
I was thinking of ArmVirtPkg, but that doesn't make sense here. Updated